extension/src/testConfigManager.ts

```diff
@@ -52,7 +52,7 @@ export class TestConfigManager {
         const editor = window.activeTextEditor;
         return this.loadConfig().then(() => {
             return workspace.openTextDocument(this._configPath).then((doc) => {
-                return window.showTextDocument(doc, editor.viewColumn);
+                return window.showTextDocument(doc, editor ? editor.viewColumn : undefined);

```

Comment:
For comment, it should clearly define what this is about, but not number or links. Indirect reference will make things unclear.